### PR TITLE
CLA & Country update

### PR DESCRIPTION
**Pull request by badbole**
Description of the issue/feature this PR addresses:
Signed CLA company and private + added Kosovo country to list
Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/cla/corporate/dajmi5.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Davor Bojkić bole@dajmi5.com https://github.com/bad_bole
+
+List of contributors:
+
+DAJ MI 5 dajmi5@gmail.com https;//github.com/dajmi5
```
Comment (mart-e) [inline doc/cla/corporate/dajmi5.md]: `: ` instead of `; `

File: doc/cla/individual/badbole.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Croatia, 2015-10-31
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
```
Comment (mart-e) [inline doc/cla/individual/badbole.md]: no need, if you are signing the corporate CLA, no need to sign the individual one

File: doc/cla/individual/stef090.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Croatia, 2017-10-31
+
```
Comment (mart-e) [inline doc/cla/individual/stef090.md]: no need, if you are signing the corporate CLA, no need to sign the individual one

**Comment by mart-e** [comment]
As the Kosovo is not yet officially recognized by the UN as an undisputed state, rejecting the addition of the country.
Closing the PR as no update for CLA too.

**Comment by mowny** [comment]
As the Kosovo is de-facto recognized by the international postal and telecommunications organizations, and thus the change is necessary to enter kosovarian customers' addresses and telephone numbers, this is extremely unprofessional behaviour, mart-e.

**Comment by mart-e** [comment]
@mowny as discussed at https://github.com/jvOrsouw/Odoo11/pull/1 we agreed to add Kosovo, just waiting for a PR to add it. Feel free to submit one.

**Comment by mart-e** [comment]
but we need somebody to [sign the CLA](https://www.odoo.com/sign-cla) to accept the request.
You can always set another person as [co-author](https://help.github.com/articles/creating-a-commit-with-multiple-authors/) if you wish to keep attribution.

**Comment by mowny** [comment]
You have already accepted this author's corporate CLA in #23602.

**Comment by mart-e** [comment]
Thank you, you are right, I don't follow all the CLA we merge.
As the CLA was already merged, no issue to merge the original patch. Done at 09ad5606604

**Comment by badbole** [comment]
TNX GUYS!

**Comment by mowny** [comment]
Okay, so now I have an updated res_country_data.xml. But how do I get odoo to update its database? The file apparently is only read at database creation, and then never again.

Are there any ugly surprises waiting for me when I use
`insert into res_country (id, currency_id, code, name, create_uid, address_format, write_date, create_date, write_uid, phone_code) values ('254', '1', 'XK', 'Kosovo', '1', E'%(street)s\n%(street2)s\n%(city)s %(state_code)s %(zip)s\n%(country_name)s', (now() at time zone 'UTC'), (now() at time zone 'UTC'), '1', '383');`
?

**Comment by mart-e** [comment]
You can update the base module to fetch the new data (beware, can be slow and will erase changes done in some records like views)

**Comment by mowny** [comment]
That's ... not really feasible. The content of that file might not change often, but when it does (e.g. a country joining/leaving € zone, Brexit ...), _all_ installations will need to be updated, preferably _not_ in a way that kills any customizations.

(While we're at it, why is the country group at the bottom of the file called "Europe", shouldn't it rather be "European Union"?)

So with
`insert into ir_model_data (create_uid, create_date, write_date, write_uid, noupdate, name, date_init, date_update, module, model, res_id) values (1, (now() at time zone 'UTC'), (now() at time zone 'UTC'), 1, 't', 'xk', '2018-09-04 14:44:12.20301', '2018-09-04 14:44:12.20301', 'base', 'res.country', 254);`
I should be good?

**Pull request closed without merging**